src/model/ident_env.cppo.ml

```diff
@@ -19,6 +19,7 @@ open Names
 
 module Id = Paths.Identifier
 module P = Paths.Path
+open Typedtree
```

Comment:
This is only used for the new `read_pattern` function. I suggest to add a `let open Typedtree in` there and to remove this `open` for consistency.